detector/detector.go

```diff
@@ -263,17 +217,27 @@ func (d *Detector) evalToStringTokens(t token.Token) ([]token.Token, error) {
 	return tokens, nil
 }
 
-func (d *Detector) isSkip(deepCheck bool, target string) bool {
+func (d *Detector) isSkip(deepCheck bool, targetType string, target string) bool {
 	if deepCheck && !d.Config.DeepCheck {
 		return true
 	}
 
-	targetResources := 0
-	for _, template := range d.Templates {
-		targetResources += len(template.Node.(*ast.ObjectList).Filter("resource", target).Items)
+	targets := 0
+	switch targetType {
+	case "resource":
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "resource" found in: detector.go:152:7 test_helper.go:71:7

<!-- Comment by [SideCI](https://sideci.com) -->